### DISCUSSION TITLE
The repository fulplan has been retired and cannot #1718600

### DISCUSSION THREAD

**Discussion by fulplan** [community/community#27591] [Other Feature Feedback, Questions, & Ideas]
Account Name: fulplan https://github.com/fulplan

Github Account Name : fulplan fulplan (FulPlan Mutasim) · GitHub**

Actually, I recently changed my username from fulplan to a different name, and I also changed my special repo name on GitHub.

After some days I started finding trouble as all because everywhere I used my previous username So I changed again to my previous one, but there is an error on GitHub Repository name. So I accidentally rename my special repository.
Now when I try to make new repo there error message pop up
‘The repository fulplan has been retired and cannot be reused.’ my tickets #1718600

Can you please fix this asap?

Regards,

Mutasim Abubakar

**Comment by jfullstackdev** [comment]
just want to comment too, 

remember that changing your username is not simply like that, it will make repos retired, treat your username as your ID that should not be changed from time to time

you are still allowed to do that, me, I think, I changed my name 3x but really the side-effects are tremendous especially redirects, actually one tricky is that, if that username after some time will be taken by another user and the user creates the same repo name, the other one will not be redirecting to the original repo, but the latest one

**Comment by airtower-luna** [comment]
You mentioned filing a ticket, so you already did the right thing. :slightly_smiling_face:  I'm afraid you'll have to wait for support staff to get to your ticket.

**Reply by fulplan** [reply]
Thank you.
